src/main/java/org/sagebionetworks/bridge/models/surveys/SurveyQuestionOption.java

```diff
@@ -12,18 +12,21 @@
     private final String detail;
     private final String value;
     private final Image image;
+    private final Boolean exclusive;
     
     @JsonCreator
     public SurveyQuestionOption(@JsonProperty("label") String label, @JsonProperty("detail") String detail,
-        @JsonProperty("value") String value, @JsonProperty("image") Image image) {
+            @JsonProperty("value") String value, @JsonProperty("image") Image image,
+            @JsonProperty("exclusive") Boolean exclusive) {
```

Comment:
I noticed there isn't a JSON serialization test in SurveyQuestionOptionTest. Should we add one?

Reply:
Added